### PR TITLE
Drop Carbon support

### PR DESCRIPTION
**Pull request by StykMartin**
Carbon is no longer used anywhere.

This was the solution before we start using Grafana / Prometheus.
That is the way. I will add further support for Prometheus in the future.

Signed-off-by: Martin Styk <mart.styk@gmail.com>

### CODE REVIEW

File: .stickler.yml [hunk 1]
```diff
@@ -7,8 +7,8 @@ linters:
     # E203: Whitespace before ':'
     # W503: Line break occurred before a binary operator
     ignore: "E203,W503"
-  black:
-    fixer: false
+  #black:
+  #  fixer: false
```
Comment (renanrodrigo) [inline .stickler.yml:11]: is black bringing more problems than expected?

**Review by renanrodrigo — APPROVED** [review]
lgtm overall

**Comment by StykMartin** [comment]
Black doesn't work as we expected. It is bragging about whole files

**Comment by JohnVillalovos** [comment]
> Black doesn't work as we expected. It is bragging about whole files

I thought it might, but nobody listened to me 😂

**Pull request merged**